holoviews/operation/datashader.py

```diff
@@ -1725,3 +1725,78 @@ class directly_connect_edges(_connect_edges, connect_edges):
 
     def _bundle(self, position_df, edges_df):
         return connect_edges.__call__(self, position_df, edges_df)
+
+
+class inspect_sample(LinkableOperation):
+    """
+    Given datashaded aggregate (Image) output, return a single
+    (hoverable) point at the sample closest to the cursor.
+    """
+
+    pixels = param.Integer(default=3, doc="""
+       Number of pixels in data space around the cursor point to search
+       for hits in (with mask_shape). The hit in this mask that is
+       closest to the cursors position is displayed.""")
+
+    mask_shape = param.ObjectSelector(default='square', objects=['square'], doc="""
```

Comment:
Maybe remove this, hard code it to a square, leave changing the shape for future work, then never do it? :-)

Reply:
Sure, in which case I would file an issue to consider other mask shapes once this is merged (as a feature request).

Reply:
Addressed in 022680df0. Will now be a feature request issue (if it seems worthwhile).